torch/storage.py

```diff
@@ -65,7 +65,7 @@ def _shared_decref(self) -> T: ...  # noqa: E704
     def _write_file(self, *args, **kwargs): ...  # noqa: E704
     def resize_(self, size: int): ...  # noqa: E704
     def _weak_ref(self, *args, **kwargs) -> T: ...  # noqa: E704
-    def is_pinned(self) -> bool: ...  # noqa: E704
+    def is_pinned(self, device="cuda") -> bool: ...  # noqa: E704
```

Comment:
Could you add documentation for this function? You can add it `torch/_storage_docs.py`, and base it off of the entry for `from_file`. Then also it would be a good idea to update the documentation for `TypedStorage.is_pinned` to match it

Reply:
Although if we do get rid of `THPStorage_isPinned` as I suggested, the docstring would just go here